css/styles.css

```diff
@@ -0,0 +1,27 @@
+
+@import "_fonts.css";
+@import "_header.css";
+
+body{
+    background: #f0f0f0;
+}
+h1.logo {
```

Comment:
It's a better practice to not use tags in CSS at all. So if tomorrow you change your `.logo` element in the HTML to `<h2>` you don't also need to change it in the CSS. Also, the CSS Class is specific enough.